config/application.rb

```diff
@@ -32,13 +32,13 @@ class Application < Rails::Application
     # Only loads a smaller set of middleware suitable for API only apps.
     # Middleware like session, flash, cookies can be added back manually.
     # Skip views, helpers and assets when generating a new resource.
-    config.api_only = true
+    config.api_only = false
     config.sso = config_for(:sso)
 
     Rails.application.config.hosts = [
       "localhost",
       "127.0.0.1",
-      "/(.*\.|)openstax\.org/",
+      /(.*\.|)openstax\.org/,
```

Comment:
did you intentionally remove the quotes around this?

Reply:
@mwharrison Yes, it did not work with the quotes.